cypress/integration/userCanSignUp.spec.js

```diff
@@ -51,6 +51,6 @@ describe('User can sign up', () => {
       cy.get('#password-confirmation').type('wrong_password')
     })
     cy.get('#submit-signup-form').click()
-    cy.get('#error-message').should('contain', 'Invalid credentials')
+    cy.contains('Invalid credentials')
```

Comment:
why did you remove `cy.get('#error-message')`?

Reply:
I've inspected the element and tried bunch of different ways but for some reason Invalid Credentials cant be found inside of the #error-message even though it is displayed.